test/data/dist/case3_balanced_pv_2.dss

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
It might be better to reduce the number of test files by including only the most complex one, and reducing it by deleting components after loading. This would also reduce the number of reads from the disk.

Reply:
@pseudocubic I removed the larger test cases, and kept the ones that we need to test the code